src/shared/credentials/accountId.ts

```diff
@@ -26,3 +27,27 @@ export async function getAccountId(credentials: AWS.Credentials, region: string)
         return undefined
     }
 }
+
+/**
+ * Temporary workaround for missing credentials bug.
+ * TODO: Remove this function when migrated over to V3 of the SDK
```

Comment:
nit: just for precision :) 
```suggestion
 * TODO: Remove this function after migrating to https://github.com/aws/aws-sdk-js-v3
```